lib/manager/poetry/extract.spec.ts

```diff
@@ -1,6 +1,9 @@
 import { readFileSync } from 'fs';
+import { fs } from '../../../test/util';
 import { extractPackageFile } from './extract';
 
+jest.mock('../../util/fs');
```

Comment:
Adding this mock here causes the ` ● .updateArtifacts() › passes private credential environment vars` test to fail. I've got *no* clue about Jest or Jest mocking so I really have no idea what is up with that.